src/poprox_recommender/handler.py

```diff
@@ -45,3 +63,48 @@ def generate_recs(event, context):
 
     logger.info("Finished.")
     return response
+
+def extract_general_topic(article):
+    news_topics = [mention.entity.name for mention in article.mentions]
+    article_topics = set()
+    for topic in news_topics:
+        if topic in general_topics:
+            article_topics.add(topic)
+    return article_topics
```

Comment:
This looks like it's doing set intersection on lists—correctly—but there's an easier way:
```python
general_topics = set("US News", ...)

def extract_general_topics(article: Article):
    article_topics = set([mention.entity.name for mention in article.mentions])
    return article_topics.intersection(general_topics)
```

Reply:
Wow the simplification of code! TIL